### PR TITLE
Remove UUID from instance names

### PR DESCRIPTION
**Pull request by jawang35**
## Purpose
https://github.com/Surfline/docker-run-on-ec2/issues/9

Removed UUID from EC2 instance `Name` tag. UUID is still appended to key pair name.

## Cleanup
- Added a `.envrc` file to make development easier.
- Added a proper logger instead of using `print`.
- Moved main code into a function `main`.

### CODE REVIEW

File: lib/ssh.py [hunk 1]
```diff
@@ -23,7 +27,7 @@ class SSH():
     :type private: str
     """
 
-    def __init__(self, host, user, private_key):
+    def __init__(self, host: str, user: str, private_key):
```
Comment (slink1993) [inline lib/ssh.py]: missing typing annotation for `private_key`

Comment (jawang35) [reply lib/ssh.py]: @stephenlink1993 what's the best way to include type annotations for complex types that are defined by third party libraries?

Comment (slink1993) [reply lib/ssh.py]: @jawang35 oh is `private_key` not type `str` in this function parameter?

Comment (slink1993) [reply lib/ssh.py]: @jawang35 and good question! typically i search to see if there is a type annotation library for the 3rd party library. as an example, i used the library:
https://pypi.org/project/boto3-type-annotations/
for boto3 in `check-model-ready`. 
If there is not a typing annotation library, you can actually do something like this:
```
def __init__(private_key: RSAKey):
    self.private_key = private_key
```
^ `mypy` will complain about this, but i think its fine in this case 👍

**Review by slink1993 — DISMISSED** [review]
minor comment, otherwise looks good to me 👍

**Pull request merged**